app/javascript/components/project-submissions/components/__tests__/modal.test.js

```diff
@@ -0,0 +1,48 @@
+import { fireEvent, render } from "@testing-library/react";
+import React from "react";
+import Modal from "../modal";
+
+describe("Modal", () => {
+  test("Checks for the specified classname on modal when the modal is not shown", () => {
+    const { getByLabelText } = render(
+      <Modal 
+        handleClose={() => {}} 
+        show={false}
+      >
+        {" "}
+      </Modal>
+    );
+
+    expect(getByLabelText("modal--hidden")).toHaveClass("react-modal--hidden");
+  });
+
+  test("Checks for the specified classname on modal when the modal is shown", () => {
+    const { getByLabelText } = render(
+      <Modal 
+        handleClose={() => {}} 
+        show={true}
+      >
+        {" "}
+      </Modal>
+    );
+
+    expect(getByLabelText("modal--shown")).toHaveClass("react-modal");
```

Comment:
Same here: Add an assertion that tests for the presence/absence of children, per line 14 of the component.